index.html

```diff
@@ -28,9 +28,152 @@ <h4 class="connect"> LET’S CONNECT</p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
+    
+   </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (2).png" alt=""></a>
+    
+        <h3  class="tonic" >Multi-Post
+            Stories</h3>
+    
+    <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
+    
+   </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (3).png" alt=""></a>
+    
+        <h3  class="tonic" >Tonic</h3>
+    
+    <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
+    
+   </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" href="#"><img src="./image/Snapshoot Portfolio1 (4).png" alt=""></a>
+    
+        <h3  class="tonic" >Multi-Post
+            Stories </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+
+    <div class="works2">
+        <div>html</div>
+        <div>css</div>
+        <div>javascript</div>
+   </div>
+   <div class="buttondir">
+    <button class="buttonProject"  type="button" class="resume-btn">See Project</button>
+    
+   </div>
+</div>
+  </section>
+  <section class="Workssection">
+    <div class="cardBack2">
+        <h3  class="tonic" >About Myself </h3>
+        <div>
+    <ul class="works">
+        <li class="canopy"> CANOPY</li>
+        <li class="BackendDev">BackendDev</li>
+        <li class="num"> 2015 </li>
+     </ul>
```

Comment:
- [ ] This appears not to be on the Figma design. I would recommend sticking to the Figma template.

### Your Design
![image](https://user-images.githubusercontent.com/86200356/199234199-4580eecd-eada-4db9-8817-aee13d3ac8d4.png)

### Figma Design
![image](https://user-images.githubusercontent.com/86200356/199234386-f099390c-ce2f-4b0e-97b3-f95705c38a24.png)